Voice Note/Models/VoiceNote.xcdatamodeld/VoiceNote.xcdatamodel/contents

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21754" systemVersion="22E261" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="Recording" representedClassName="Recording" syncable="YES" codeGenerationType="class"/>
+    <entity name="VoiceNoteEntity" representedClassName="VoiceNoteEntity" isAbstract="YES" syncable="YES" codeGenerationType="class">
+        <attribute name="createdAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="duration" optional="YES" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
+        <attribute name="id" optional="YES" attributeType="UUID" usesScalarValueType="NO"/>
+        <attribute name="latitude" optional="YES" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
+        <attribute name="longitude" optional="YES" attributeType="Double" defaultValueString="0.0" usesScalarValueType="YES"/>
```

Comment:
Should not it supposed to be CLLocation instead of latitude and longitude? Then it would be easy to integrate to MapKit. 
We can use public init(latitude: CLLocationDegrees, longitude: CLLocationDegrees) initializer of CLLocation class to create a new CLLocation object. CLLocationDegrees is Double (typealias)

What do you guys think @giaongo @momras201 @TaiNguyen2407 @darjapo

Reply:
@anwar-ulhaq No need to have any extra function to load audio files from folder. We just need to load the audio file when user clicks on the play button of RecordingCardView.

Reply:
@anwar-ulhaq  Agree to your suggestion. Having CLLocationDegrees would be much more convenient since we dont need to re-transform them from lat and long to CLLocationDegrees again.

Reply:
@anwar-ulhaq Agree. It will be much easier to work with the map related features the way you suggested.